tests/main/abort/task.yaml

```diff
@@ -1,5 +1,8 @@
 summary: Check change abort
 
+# Disabled for Fedora until test case is properly ported
+systems: [-fedora-*]
```

Comment:
That the `tests/main` suite is disabled is actually a mistake. My bad that I tested locally only with an explicit `linode:fedora-25-64:tests/main/null`. However, we can't disable all test cases in the beginning for Fedora as we need at least one for spread to prepare the system. Otherwise it will simply find no matches for the system and don't bother with it at all. That is why I added a `tests/main/null` test case and disabled all others.

Also a disabled `tests/main` suite with a single sub test enables still gives
```
$ spread -v -debug -reuse -resend linode:fedora-25-64:tests/main
2017/05/30 07:05:34 Found /home/simon/Work/ubuntu/snappy/work/src/github.com/snapcore/snapd/spread.yaml.
error: nothing matches provider filter
```